### PR TITLE
Timespinner: Enter Sandman flag

### PR DESCRIPTION
**Pull request by Jarno458**
Say your prayers, little one, don't forget, my son
To include everyone
I tuck you in, warm within, keep you free from sin
Til the Sandman, he comes

Sleep with one eye open
Gripping your pillow tight

Exit light
Enter night
Take my hand
We're off to never-never land

Something's wrong, shut the light, heavy thoughts tonight
And they aren't of Snow White
Dreams of war, dreams of liars, dreams of dragons' fire
And of things that will bite, yeah

leep with one eye open
Gripping your pillow tight

Exit light
Enter night
Take my hand
We're off to never-never land
Yeah-hah

Now, I lay me down to sleep (Now, I lay me down to sleep)
Pray the Lord my soul to keep (Pray the Lord my soul to keep)
If I die before I wake (If I die before I wake)
Pray the Lord my soul to take (Pray the Lord my soul to take)
Hush, little baby, don't say a word
And never mind that noise you heard
It's just the beasts under your bed
In your closet, in your head

## What is this fixing or adding?
The new Enter Sandman flag as a option for your timespinner yaml

## How was this tested?
Generated 20 seeds with and without the flag and checked spoilers logs
Played through one game where it was enabled to see if it was properly passed over from server to client

### CODE REVIEW

File: worlds/timespinner/Options.py [hunk 1]
```diff
@@ -292,18 +277,20 @@ class ShowDrops(Toggle):
     display_name = "Show Bestiary Item Drops"
 
 
+class EnterSandman(Toggle):
+    "Ancient Pyramid is unlocked by the Twin Pyramid Keys, but the final boss door opens if you have all 5 Timespinner pieces"
```
Comment (Alchav) [inline worlds/timespinner/Options.py:281]: Suggestion:
"The Ancient Pyramid is unlocked by the Twin Pyramid Key, but the final boss door only opens if you have all 5 Timespinner pieces"

Comment (Jarno458) [reply worlds/timespinner/Options.py]: Fine

**Comment by Berserker66** [comment]
You're renaming one Location, which needs incrementing of data version

**Comment by Jarno458** [comment]
> You're renaming one Location, which needs incrementing of data version

No I am not, I am adding 1 new region, and renamed 1 region, the changes are only in the logic (regions/requirements) not in location or item name/ids

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**